CHANGELOG.md

```diff
@@ -7,17 +7,19 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
-## [3.6.1] - 2021-02-23
-
-### Removed
-
-- Do not add `VersionBundle` to new `CertConfig` specs (`CertConfig`s are now versioned using a label). **This change requires using `cert-operator` 1.0.0 or later.**
+## [3.7.0] - 2021-03-15
 
 ### Added
 
 - Create app CR for per cluster app-operator instance.
 - Add `appfinalizer` resource to remove finalizers from workload cluster app CRs.
 
+## [3.6.1] - 2021-02-23
+
+### Removed
+
+- Do not add `VersionBundle` to new `CertConfig` specs (`CertConfig`s are now versioned using a label). **This change requires using `cert-operator` 1.0.0 or later.**
```

Comment:
I messed up resolving the merge conflict. 🤦‍♂️ 

See https://github.com/giantswarm/cluster-operator/blob/v3.6.1/CHANGELOG.md 

v3.6.1 includes the certconfig changes from @stone-z and not my app-operator changes.